tests/fast-integration/telemetry-test/telemetry-test.js

```diff
@@ -179,4 +179,14 @@ describe('Telemetry Operator tests, prepare the environment', function() {
     const logsPresent = await logsPresentInLoki(labels, testStartTimestamp);
     assert.isTrue(logsPresent, 'No kyma-system logs present in Loki');
   });
+
+  it('Should verify that Kubernetes annotations are not pushed', async () => {
+    const labels = '{namespace="mockserver", container="mockserver"}';
+    const responseBody = await queryLoki(labels, testStartTimestamp);
+    assert.isTrue(responseBody.data.result.length > 0, 'No mockserver logs present in Loki');
+    const entry = JSON.parse(responseBody.data.result[0].values[0][1]);
+
+    expect(entry.log).not.to.have.string('annotations');
+    expect(entry.log).to.have.string('labels');
+  });
```

Comment:
Let's maybe create 2 acceptance tests: one for `KeepAnnotations` and one for `DropLabels`?

Reply:
don't want to create an extra LogPipeline for this, will try to reuse the Loki LogPipeline

Reply:
⬆️ won't work as we remove all kubernetes keys in the Loki LogPipeline, see https://github.com/kyma-project/kyma/blob/main/resources/logging/charts/loki/templates/kyma-additions/logpipeline_loki.yaml#L16